docs-2.0/nebula-explorer/graph-explorer/node-filtering.md

```diff
@@ -1,4 +1,4 @@
-# Vertex Filter 
+# Vertex filter 
 
 The Vertex Filter helps you filter the vertices and edges displayed on the canvas. You can filter data by tag only or by one or more sets of filter conditions. 
```

Comment:
Maybe you need to make the context consistent? 'Vertex filter' please

Reply:
> Maybe you need to make the context consistent? 'Vertex filter' please

Babe, Vertex Filter is a UI word.  We can keep it when meaning a UI word or the name of a feature.
![image](https://user-images.githubusercontent.com/78209557/177241756-a24dfba2-2ff9-487f-993b-f50293e4fdf1.png)